nemo/collections/tts/parts/utils/tts_dataset_utils.py

```diff
@@ -23,6 +23,17 @@
 from torch.special import gammaln
 
 
+def get_audio_paths(audio_path, base_path):
+    if os.path.isabs(audio_path):
+        abs_path = audio_path
+        rel_path = audio_path.relative_to(base_path)
```

Comment:
`relative_to` is a function for `pathlib.Path` object. Could you pls add type hints in the function?